src/Microsoft.ML.Data/DataLoadSave/Text/TextLoader.cs

```diff
@@ -343,28 +344,28 @@ public class ArgumentsCore
                     " missing value and an empty value is denoted by \"\". When false, consecutive separators" +
                     " denote an empty value.",
                 ShortName = "quote")]
-            public bool AllowQuoting = true;
+            public bool AllowQuoting = DefaultArguments.AllowQuoting;
 
             [Argument(ArgumentType.AtMostOnce, HelpText = "Whether the input may include sparse representations", ShortName = "sparse")]
-            public bool AllowSparse = true;
+            public bool AllowSparse = DefaultArguments.AllowSparse;
 
             [Argument(ArgumentType.AtMostOnce,
                 HelpText = "Number of source columns in the text data. Default is that sparse rows contain their size information.",
                 ShortName = "size")]
             public int? InputSize;
 
             [Argument(ArgumentType.AtMostOnce, Visibility = ArgumentAttribute.VisibilityType.CmdLineOnly, HelpText = "Source column separator. Options: tab, space, comma, single character", ShortName = "sep")]
-            public string Separator = "tab";
+            public string Separator = "tab"; //DefaultArguments.Separator
```

Comment:
>DefaultArguments [](start = 47, length = 16)

Is this comment needed?

Reply:
or maybe just two default arguments Separator and SeparatorChars? Two different params doesnt guarantee things to be in sync. Im assuming since these are argruments that they can be set by the user -- so what happens if a user specifies Separator to be space, but doesnt set the separator chars?

---
In reply to: [243411408](https://github.com/dotnet/machinelearning/pull/1878#discussion_r243411408) [](ancestors = 243411408)